spec/models/user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should have_valid(:first_name).when("Jimmy", "Linc") }
+  it { should_not have_valid(:first_name).when(nil, "") }
+
+  it { should have_valid(:last_name).when("Ranger", "Kupke") }
+  it { should_not have_valid(:last_name).when(nil, "") }
+
+  it { should have_valid(:email).when("example@gmail.com", "erlich@aviato.net") }
+  it { should_not have_valid(:email).when("example", "example@", "exam@gma", "example.com", nil) }
+
+  describe "#admin?" do
+    context "when user is a admin" do
+      it "should give back true if user an admin" do
```

Comment:
More explicit terminology would be 'will return true'. We're not testing for what your code _should_ be doing, we're testing what it _will_ do. A 'return value' is a specific, defined programming term, where as 'give back' is more vague and could mean a number of things